lib/expansion_rules.rb

```diff
@@ -57,7 +57,7 @@ def recurring
   CUSTOM_EXPANSION_FIELDS = [
     { document_type: :redirect,                   fields: [] },
     { document_type: :gone,                       fields: [] },
-    { document_type: :contact,                    fields: DEFAULT_FIELDS_WITH_DETAILS },
+    { document_type: :contact,                    fields: DEFAULT_FIELDS + [%i(details description), %i(details title), %i(details contact_form_links), %i(details post_addresses), %i(details email_addresses), %i(details phone_numbers)] },
```

Comment:
Since we're likely to be doing this a lot, I wonder if we could make a convenience function that given a list of fields returns a list of `%i(details <field>)`:

```
DEFAULT_FIELDS + details_fields(:description, :title, :contact_form_links, ...)
```